client_test.go

```diff
@@ -1021,7 +1021,7 @@ func TestClient_GenerateTenantToken(t *testing.T) {
 			args: args{
 				UID: "TestGenerateTenantTokenWithoutApiKey",
 				client: NewClient(ClientConfig{
-					Host:   "http://localhost:7700",
+					Host:   meilisearchHost,
```

Comment:
Are constants written in capslock in go ? `MEILISEARCH_HOST` ?

Reply:
Do you mean global constants? No.
But thank I realize that it wasn't great at all what I done I create a function instead 😃